common/src/main/java/com/wynntils/functions/CombatXpFunctions.java

```diff
@@ -76,10 +73,10 @@ public String getValue(FunctionArguments arguments) {
         }
     }
 
-    public static class XpRawFunction extends Function<Float> {
+    public static class XpRawFunction extends Function<Double> {
         @Override
-        public Float getValue(FunctionArguments arguments) {
-            return Models.CombatXp.getCurrentXp();
+        public Double getValue(FunctionArguments arguments) {
+            return (double) Models.CombatXp.getCurrentXp();
```

Comment:
Is this cast really needed?

Reply:
Yes, I would rather have all functions return a double, but the combat xp model is heavily using floats at the moment. I would rather not change every float to double, or do you want me to?

Reply:
No, don't change type. I thought the value could be widened automatically but maybe that does not work when it also needs boxing.